### PR TITLE
CAKE-5230 | Add new variables and use them/send them to mobile-wiki

### PR DESCRIPTION
**Pull request by vforge**
- Create new `wgWatchShowGeos`, `wgWatchShowTrackingLabel` and `wgWatchShowEnabledDate` and use them in Watch Show module on MW (and add them wo wf variables for feeds and mobile-wiki)
- Create new `wgAffiliateUnitEnabledDate` and send that to mobile-wiki

### CODE REVIEW

**Comment by vforge** [comment]
Pinging @Wikia/cake 
FYI @Wikia/iwing

File: extensions/wikia/MercuryApi/models/MercuryApi.class.php [hunk 1]
```diff
@@ -172,7 +172,8 @@ public function getMobileWikiVariables() {
 		global $wgCityId, $wgStyleVersion, $wgContLang, $wgContentNamespaces, $wgDefaultSkin, $wgCdnRootUrl,
 		       $wgRecommendedVideoABTestPlaylist, $wgFandomAppSmartBannerText, $wgTwitterAccount,
 		       $wgEnableFeedsAndPostsExt, $wgEnableEmbeddedFeeds, $wgDevelEnvironment, $wgQualarooDevUrl, $wgQualarooUrl,
-		       $wgRightsText, $wgRightsUrl, $wgRedirectFilePagesForAnons, $wgEnableDiscussions;
+		       $wgRightsText, $wgRightsUrl, $wgRedirectFilePagesForAnons, $wgEnableDiscussions,
+		       $wgAffiliateUnitEnabledDate;
```
Comment (rybmat) [inline extensions/wikia/MercuryApi/models/MercuryApi.class.php:176]: what about `wgWatchShowGeos` and `wgWatchShowEnabledDate`? according to PR description both of them should be here as well

**Review by rybmat — APPROVED** [review]

File: extensions/wikia/MercuryApi/models/MercuryApi.class.php (hunk 1 above)
Comment (vforge) [reply extensions/wikia/MercuryApi/models/MercuryApi.class.php:176]: @rybmat Those are exported via hook with this: https://github.com/Wikia/app/blob/cake-5230/extensions/wikia/WatchShow/WatchShowHooks.class.php#L11-L12

File: includes/wikia/VariablesBase.php [hunk 2]
```diff
@@ -8939,6 +8939,17 @@
...
+/**
+ * Watch Show extension default values
+ */
+$wgWatchShowEnabledDate = '2099-12-31T00:00:00Z'; // ISO-8601
+
+/**
+ * Affiliate units
+ * @var string
+ */
+$wgAffiliateUnitEnabledDate = '2099-12-31T00:00:00Z'; // ISO-8601
```
Comment (mklucsarits) [inline includes/wikia/VariablesBase.php:8951]: Remember to update this code before 2099.

**Review by mklucsarits — APPROVED** [review]

**Pull request merged**